pipeline/lab.py

```diff
@@ -0,0 +1,409 @@
+import datajoint as dj
+import numpy as np
+from . import get_schema_name
+
+schema = dj.schema(get_schema_name('lab'))
+
+
```

Comment:
External collaborators from other labs (esp. UCSD) may contribute data. 
Add 
```suggestion
@schema
class Lab(dj.Manual): 
    definition = """ # Lab
    lab : varchar(255)  #  lab conducting the study
    ----
    institution  : varchar(255)  # Institution to which the lab belongs
    """
```